keyboards/gopolar/gg86/keymaps/bongocat/oled/bongocat.h

```diff
@@ -0,0 +1,376 @@
+/* Copyright 2021 HorrorTroll <https://github.com/HorrorTroll>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+// WPM-responsive animation stuff here
+#define IDLE_FRAMES 5
+#define IDLE_SPEED 10  // below this wpm value your animation will idle
+// #define PREP_FRAMES 1 // uncomment if >1
+#define TAP_FRAMES 2
+#define ANIM_WPM_LOWER 20  // above this wpm value typing animation to trigger
+#define ANIM_FRAME_DURATION_MAX 450 // longest animation duration in ms
+#define ANIM_FRAME_DURATION_MIN 100 // shortest animation duration in ms
+#define IDLE_FRAME_DURATION 300  // how long each frame lasts in ms
+#define ANIM_FRAME_RATIO 2.5 // how aggressively animation speeds up with wpm
+// #define SLEEP_TIMER 60000 // should sleep after this period of 0 wpm, needs fixing
+#define ANIM_SIZE 525  // number of bytes in array, minimize for adequate firmware size, max is 1024
+#define MAX(x, y) (((x) > (y)) ? (x) : (y)) // Math.max macro
+
+uint32_t curr_anim_duration = 0; // variable animation duration
+uint32_t bongo_timer = 0;
+uint32_t bongo_sleep = 0;
+uint8_t  current_idle_frame = 0;
+// uint8_t current_prep_frame = 0; // uncomment if PREP_FRAMES >1
+uint8_t current_tap_frame = 0;
+
+// Code containing pixel art, contains:
+// 5 idle frames, 1 prep frame, and 2 tap frames
+
+// To make your own pixel art:
+// save a png/jpeg of an 128x32 image (resource: https://www.pixilart.com/draw )
+// follow this guide up to and including "CONVERT YOUR IMAGE" https://docs.splitkb.com/hc/en-us/articles/360013811280-How-do-I-convert-an-image-for-use-on-an-OLED-display-
+// replace numbers in brackets with your own
+// if you start getting errors when compiling make sure you didn't accedentally delete a bracket
+static void render_bongocat(void) {
```

Comment:
This is still going to run into the same issues, potentially, as you've just renamed the c file to an header file, basically. 

This really should be included with `SRC += bongocat.c` to be properly compiled. 

But it will probably need `#include "quantum.h"` for simplicity.

Reply:
![image](https://user-images.githubusercontent.com/24840279/163184811-0204bff5-92b7-47db-a635-cf55a53c7c21.png)

Ughhh, idk what to do. Somehow doing that seem like it couldn't find that function 😭

Reply:
You need a prototype for it.  Eg, a bongocat.h that has `void renger_bongocat(void);` in it, and that included.

Reply:
Thanks, it solved the problem now